new-site/src/docs/elements/design-tokens/colour/components/brand-colours-table.tsx

```diff
@@ -0,0 +1,34 @@
+  { colourNameEn: 'Fog', colourNameFi: 'Sumu', baseColourExample: <ColorExample color="var(--color-fog)" name="fog base brand colour example" />, darkVariantExample: <ColorExample color="var(--color-fog-dark)" name="fog dark variant example" />, mediumLightVariantExample: <ColorExample color="var(--color-fog-medium-light)" name="fog medium light example" />, lightVariantExample: <ColorExample color="var(--color-fog-light)" name="fog light variant example" />},
+  { colourNameEn: 'Metro', colourNameFi: 'Metro', baseColourExample: <ColorExample color="var(--color-metro)" name="metro base brand colour example" />, darkVariantExample: <ColorExample color="var(--color-metro-dark)" name="metro dark variant example" />, mediumLightVariantExample: <ColorExample color="var(--color-metro-medium-light)" name="metro medium light example" />, lightVariantExample: <ColorExample color="var(--color-metro-light)" name="metro light variant example" />},
+  { colourNameEn: 'Summer', colourNameFi: 'Kesä', baseColourExample: <ColorExample color="var(--color-summer)" name="summer base brand colour example" />, darkVariantExample: <ColorExample color="var(--color-summer-dark)" name="summer dark variant example" />, mediumLightVariantExample: <ColorExample color="var(--color-summer-medium-light)" name="summer medium light example" />, lightVariantExample: <ColorExample color="var(--color-summer-light)" name="summer light variant example" />},
+  { colourNameEn: 'Suomenlinna', colourNameFi: 'Suomenlinna', baseColourExample: <ColorExample color="var(--color-suomenlinna)" name="suomenlinna base brand colour example" />, darkVariantExample: <ColorExample color="var(--color-suomenlinna-dark)" name="suomenlinna dark variant example" />, mediumLightVariantExample: <ColorExample color="var(--color-suomenlinna-medium-light)" name="suomenlinna medium light example" />, lightVariantExample: <ColorExample color="var(--color-suomenlinna-light)" name="suomenlinna light variant example" />},
+  { colourNameEn: 'Tram', colourNameFi: 'Spåra', baseColourExample: <ColorExample color="var(--color-tram)" name="tram base brand colour example" />, darkVariantExample: <ColorExample color="var(--color-tram-dark)" name="tram dark variant example" />, mediumLightVariantExample: <ColorExample color="var(--color-tram-medium-light)" name="tram medium light example" />, lightVariantExample: <ColorExample color="var(--color-tram-light)" name="tram light variant example" />},
+];
+
+export default function BrandColoursTable() {
+  return (
+    <Table dense cols={cols} rows={rows} indexKey="id" renderIndexCol={false} />
```

Comment:
This can be simplified to:

`<Table dense cols={cols} rows={rows} indexKey="colourNameEn" />`